src/lib/config/passport.ts

```diff
@@ -33,23 +30,18 @@ export function configure(
 			profile: any,
 			cb: oauth2.VerifyCallback
 		) => {
-			profile.accessToken = accessToken
-
-			try {
-				const identityDetails = await identity.getDetails(accessToken)
-				const regDetails = await registry.profile(accessToken)
+			// profile.accessToken = accessToken
```

Comment:
Don't we need this for further request to e.g. registry-service? If not delete the line rather than commenting.